src/foundry/foundry.js/application.d.ts

```diff
@@ -323,7 +323,7 @@ declare abstract class Application<Options extends ApplicationOptions = Applicat
    * @returns A promise resolving to the constructed jQuery object
    * @internal
    */
-  protected _renderOuter(): Promise<HTMLElement> | Promise<JQuery<JQuery.Node>>;
+  protected _renderOuter(): Promise<HTMLElement | JQuery<JQuery.Node>>;
```

Comment:
While we're at it, I think it should just be
```suggestion
  protected _renderOuter(): Promise<JQuery>;
```
anyways.

Reply:
🎉 I updated my original commit to keep the history clean.